### PR TITLE
fix: Remove unsupported action types

### LINKED ISSUE #1482: Remove unsupported response types from the app
**Actual Behaviour**

The RSS action and some other action types have been discontinued from the server and hence is added redundantly in the app and is causing issues while refactoring the code.

**Expected Behaviour**

Remove the redundant action as it is not supported by the server anymore.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by batbrain7**
Fixes #1482
 
Changes:  Many action types are not supported by the server, so it is removed.

### CODE REVIEW

**Comment by batbrain7** [comment]
@codedsun Please review !

File: app/src/main/java/org/fossasia/susi/ai/chat/ParseSusiResponseHelper.kt [hunk 1]
```diff
@@ -65,13 +65,6 @@ class ParseSusiResponseHelper {
                 null
             }
 
-            Constant.RSS -> datumList = try {
-                susiResponse.answers[0].data
-            } catch (e: Exception) {
-                Timber.e(e)
-                null
-            }
-
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/chat/ParseSusiResponseHelper.kt]: Is it just this?

Comment (batbrain7) [reply app/src/main/java/org/fossasia/susi/ai/chat/ParseSusiResponseHelper.kt]: @iamareebjamal the viewholders and adapters related to this action are also used by other actions so only this part needs to be removed

**Comment by batbrain7** [comment]
@arundhati please review this

**Comment by batbrain7** [comment]
@arundhati24 I checked with @hkedia321 and he told me that RSS is not supported for months now also web search is also not supported. if the code should be kept in the app for future then we should only remove the parts where it reflects some response to the app.

**Comment by arundhati24** [comment]
@batbrain7 Could you please link me to the discussion you had with @hkedia321 regarding this?

**Comment by batbrain7** [comment]
@arundhati24 you may ask him yourself, as all discussions do not take place on the issue tracker

**Comment by hkedia321** [comment]
@batbrain7 Though some advanced action types have stopped coming from the server (see [susi_server/issues/593](https://github.com/fossasia/susi_server/issues/593) and [susi_server/issues/726](https://github.com/fossasia/susi_server/issues/726)), I don't think so we should remove the support from the client sides for these. The advanced response types may come back. Also we have support for these in chat.susi.ai. @arundhati24 What do you think?

**Comment by batbrain7** [comment]
Thanks @hkedia321 for the input so I think the support should not be removed, but should not be not detected as of now in the client and can be detected later on when the server supports these

**Comment by batbrain7** [comment]
@iamareebjamal @arundhati24 please review

**Comment by arundhati24** [comment]
@hkedia321 @akshatnitd I agree with both of you and I think these should not be removed from the client for now.

**Comment by arundhati24** [comment]
@batbrain7 If the discussion has not taken place on issue tracker, it would have taken place on the Gitter channel. As I couldn't find any such discussion on the project channels, I asked you to link me to the discussion. How should I know when you had asked and what you had asked without reading the conversation. If you had asked it on a personal chat, that's a different case.

**Comment by batbrain7** [comment]
@akshatnitd @arundhati24 I have not deleted the implementation but instead aligned it with the web client if you closely look at the PR, now whenever we ask any response of the type rss or web search it shows an error message just as in case of the web client , and since the web client has more features than the android client I think the goal should be to align them together,
Also if there is no other reason then please review this PR and merge it.

**Comment by batbrain7** [comment]
@iamareebjamal please merge

**Pull request merged**